test/makeRule.js

```diff
@@ -26,7 +28,7 @@ const parserOptions = {
 
 {
   const options = [
-    { schemaJson },
+    { schemaString },
```

Comment:
i dont think you want to replace this — you should add a separate block of tests for this